src/ORM/Connect/MySQLDatabase.php

```diff
@@ -66,6 +66,8 @@ class MySQLDatabase extends Database implements TransactionManager
      */
     private $transactionManager = null;
 
+    public $transactionNesting = 0;
```

Comment:
```suggestion
    public $transactionNesting;
```
Don't set a default value. There was no default value with the dynamic variable so there shouldn't be one now either.